README.md

```diff
@@ -29,26 +32,23 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 Example module invocation:
 ```
 module "apiary-shuntingyard" {
-  source             = "git::https://github.com/ExpediaInc/apiary-replication.git?ref=master"
-  instance_name      = "shuntingyard-test"
-  aws_region         = "us-west-2"
-  vpc_id             = "vpc-1"
-  subnets            = ["subnet-1", "subnet-2"]
-  allowed_s3_buckets = ["bucket-1", "bucket-2"]
-
-  tags = {
-    Name = "Apiary-Shuntingyard"
-    Team = "Operations"
-  }
-
-  source_metastore_uri               = "thrift://ip-address:9083"
-  target_metastore_uri               = "thrift://ip-address:9083"
-  metastore_events_sns_topic         = "arn:aws:sns:us-west-2:1234567:metastore-events-sns-topic"
-  selected_tables                    = "database_1.table_1, database_2.table_2"
-  ct_common_config_yaml              = "${data.template_file.ct_common_config_yaml.rendered}"
-
-  docker_image                       = "your.docker.repo/apiary-shuntingyard"
-  docker_version                     = "latest"
+  source                      = "git::https://github.com/ExpediaInc/apiary-replication.git"
```

Comment:
ExpediaInc -> ExpediaGroup (we moved it)

Reply:
Fixed.